### PR TITLE
Add supplemental material for HTML and CSS article

### PR DESCRIPTION
**Pull request by acsany**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: html-and-css/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# HTML and CSS for Python Developers
+
+Supplementing materials for the [tutorial on GitHub Copilot](https://realpython.com/html-and-css-for-python-developers/) hosted on Real Python.
```
Comment (bzaczynski) [inline html-and-css/README.md:3]: :warning: **Fact Check**

I've been a victim of the same mistake myself: copy and paste :wink: 

```suggestion
Supplementing materials for the [tutorial on HTML and CSS for Python Developers](https://realpython.com/html-and-css-for-python-developers/) hosted on Real Python.
```

**Review by bzaczynski — APPROVED** [review]
@acsany Apart from one simple copy-paste mistake, it looks good to me :blush: 

By the way, you haven't included any Python scripts shown in the tutorial. I think it would be helpful to include them here as well!

**Comment by acsany** [comment]
Thanks for your review and for spotting the copy-paste mistake. 😬

> By the way, you haven't included any Python scripts shown in the tutorial. I think it would be helpful to include them here as well

I was thinking about adding the Python files but I refrained from that because of two reasons:

1. The scripts make sense for the tutorial but don't show best practices for creating and parsing HTML. 
2. One idea of the supplementing files is, that they can be used as a project quickstart. Some readers might be confused about the Python files in it.

But I don't feel strongly about it. If you think it would bring value to the readers, I'm happy to add them, too. 🙂

**Comment by bzaczynski** [comment]
@acsany Yeah, I think some readers will appreciate having all the Python code in one place :+1:

**Pull request merged**